src/main/java/com/ays/common/scheduler/UnusedInvalidTokenDeletionScheduler.java

```diff
@@ -0,0 +1,36 @@
+package com.ays.common.scheduler;
+
+import com.ays.auth.repository.AysInvalidTokenRepository;
+import com.ays.parameter.model.AysParameter;
+import com.ays.parameter.service.AysParameterService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+
+/**
+ * Scheduler to remove all unused invalid token.
+ */
+@Component
+@RequiredArgsConstructor
+@ConditionalOnProperty(name = "ays.scheduler.UnusedInvalidTokenDeletionScheduler.enabled", havingValue = "true", matchIfMissing = true)
+class UnusedInvalidTokenDeletionScheduler {
+
+    private final AysInvalidTokenRepository aysInvalidTokenRepository;
+    private final AysParameterService aysParameterService;
+
+    /**
+     * Clears all unused invalid tokens from the database.
+     */
+    @Scheduled(cron = "${ays.scheduler.UnusedInvalidTokenDeletionScheduler.cron}")
```

Comment:
Burada günlük çalışmasını sağlayabiliriz, parametrik olarak değiştirmemizi gerektiren bir şey olmayabilir

Reply:
Test ortamında daha kısa bir zaman verip çalıştığını test edebilmek için parametrik olarak verildi.